packaging/snapd.mk

```diff
@@ -57,7 +57,10 @@ go_binaries = snap snapctl snap-seccomp snap-update-ns snap-exec snapd
 .PHONY: all
 all: $(go_binaries) 
 
-snap snap-seccomp:
+snap:
+	go build -tags nomanagers -buildmode=pie $(import_path)/cmd/$@
```

Comment:
How does snapd.mk deal with the other potential flags like withtestkeys?

Reply:
It passes it explicitly, only when building snapd:
```
snapd:
ifeq ($(with_testkeys),1)
	go build -buildmode=pie -tags withtestkeys $(import_path)/cmd/$@
```